Purchases/Purchasing/Purchases.swift

```diff
@@ -774,8 +774,8 @@ public extension Purchases {
 
     /// Returns an `AsyncStream` of ``CustomerInfo`` changes.
     ///
-    /// - Seealso ``PurchasesDelegate/purchases(_:receivedUpdated:)
-    /// - Seealso ``Purchases/customerInfo()``
+    /// - SeeAlso: ``PurchasesDelegate/purchases(_:receivedUpdated:)``
```

Comment:
oops

Reply:
Oooh, does it have to be `SeeAlso` for it to work?

Reply:
the "oops" was because I missed the backticks at the end 😂 
it doesn't have to have the capitalized A, but it does need the semicolon after `seealso:`

Reply:
Oooh gotcha. So your comment about it still not showing up is still valid?

Reply:
to be clear, the "See Also" doesn't actually seem to generate a See Also section in the docs, when the correct syntax is used. Perhaps the archive does actually generate correctly, I need to try it out